### PR TITLE
256colors

### PR DESCRIPTION
**Pull request by jonathanabennett**
This is a simple start to fixing the 256 colors support issue raise in #2. It needs error correction and, since I have only been programming in Nim for a week, I'm still not sure of the best way to do that. But hopefully this code moves us toward 256 color support. I have tested it naively with several different color codes and it has worked successfully.

### CODE REVIEW

File: nimbox.nim [hunk 1]
```diff
@@ -168,6 +168,24 @@ proc print*[T](_: Nimbox, x, y: int, text: string,
     tbChangeCell(cast[cint](x + i), yInt, c.uint32, fgInt, bgInt)
     i.inc()
 
+proc print*[T](_: Nimbox, x, y: int, text: string,
+               fg: int, bg: int, style: T) =
```
Comment (dom96) [inline nimbox.nim:172]: Would be nice to use the ``Color`` type from the stdlib ``colors`` module here, sucks that it clashes with the ``Color`` type defined in this module though :/

**Review by dom96 — CHANGES_REQUESTED** [review]
Thank you for this.

It seems you've sadly included the changes from the other PR in this one too.

**Comment by jonathanabennett** [comment]
Is there an easy way to back out the other commit? I'm still fairly new to git and forgot to checkout back to the master branch before creating this branch.

**Comment by dom96** [comment]
Might be easiest to just re-do the PR

**Pull request closed without merging**